src/app/pages/Home/OtherComponents/MainBanner/index.tsx

```diff
@@ -134,12 +140,47 @@ const BalanceInfo: FC<{ accountPkh: string }> = ({ accountPkh }) => {
   );
 };
 
+const EvmBalanceInfo: FC<{ address: HexString }> = ({ address }) => {
+  const network = useEvmNetwork();
+  const decimals = network.nativeCurrency.decimals;
+
+  const viemClient = useMemo(
```

Comment:
please consider creating  fn like `getTezosFastRpcClient` so we could reuse this in the future